src/SprykerEco/Zed/CrefoPay/Business/Quote/Expander/Mapper/CrefoPayQuoteExpanderMapper.php

```diff
@@ -0,0 +1,368 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\CrefoPay\Business\Quote\Expander\Mapper;
+
+use ArrayObject;
+use Generated\Shared\Transfer\AddressTransfer;
+use Generated\Shared\Transfer\CrefoPayApiAddressTransfer;
+use Generated\Shared\Transfer\CrefoPayApiAmountTransfer;
+use Generated\Shared\Transfer\CrefoPayApiBasketItemTransfer;
+use Generated\Shared\Transfer\CrefoPayApiCreateTransactionRequestTransfer;
+use Generated\Shared\Transfer\CrefoPayApiPersonTransfer;
+use Generated\Shared\Transfer\CrefoPayApiRequestTransfer;
+use Generated\Shared\Transfer\ItemTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\TotalsTransfer;
+use Spryker\Shared\Shipment\ShipmentConstants;
+use SprykerEco\Service\CrefoPay\CrefoPayServiceInterface;
+use SprykerEco\Zed\CrefoPay\CrefoPayConfig;
+use SprykerEco\Zed\CrefoPay\Dependency\Facade\CrefoPayToLocaleFacadeInterface;
+use SprykerEco\Zed\CrefoPay\Dependency\Service\CrefoPayToUtilTextServiceInterface;
+
+class CrefoPayQuoteExpanderMapper implements CrefoPayQuoteExpanderMapperInterface
+{
+    protected const INTEGRATION_TYPE = 'SecureFields';
+    protected const AUTO_CAPTURE = 'false';
+    protected const CONTEXT = 'ONLINE';
+    protected const USER_TYPE_PRIVATE = 'PRIVATE';
+    protected const USER_TYPE_BUSINESS = 'BUSINESS';
+    protected const SALUTATION_MAPPING = ['Mr' => 'M', 'Ms' => 'F', 'Mrs' => 'F', 'Dr' => 'M'];
+    protected const AVAILABLE_LOCALES = ['EN', 'DE', 'ES', 'FR', 'IT', 'NL'];
```

Comment:
Where did we get list of available locales?

Reply:
We got it from CrefoPay documentation.